dist/challenge-templates/xss-bot/challenge/bot.js

```diff
@@ -50,7 +51,7 @@ if (BLOCK_SUBORIGINS) {
     socket.state = 'LOADED';
     let cookie = JSON.parse(fs.readFileSync('/home/user/cookie'));
 
-    const context = await browser.createIncognitoBrowserContext();
+    const context = await browser.createBrowserContext();
```

Comment:
Incognito is to avoid saving state. Is this not needed?

Reply:
This is replaced by --incognito in the args above. It's missing though if the BLOCK_SUBORIGINS is false

Reply:
Changed to a push() outside of the if.